DuggaSys/sectioned.php

```diff
@@ -162,30 +162,30 @@ function updateGithubDir($pdo, $githubDir, $lid) {
 
 		<div id='Sectionlist'>
 
-		<div class='course' style='display:flex; align-items:center; justify-content:flex-end; '>
+		<div id='courseHeader' class='course' >
 			
-			<!-- Undo button -->
+			<!-- Undo button 
 			<input id="undoButton" value="&#9851;" type="button" class='submit-button-newitem' title="Undo deleted example" style="position: absolute; padding-right:5px; margin-right:165px; display: none;" onclick="cancelDelete();">
-			<!-- Undo button END -->
+			 Undo button END -->
```

Comment:
You need to revert this code... When you open and close a comment with code within it, the code will also be commented, therefor this change makes runnable code not runnable anymore, since it's now a comment.

Reply:
Also fix the styling for the input here, remove inline

Reply:
the button is not used or ever apers anymore so i commented it out because i did not know if the person working on it would need it again, it is replaced by a pop-up toast